autopush-common/src/db/bigtable/mod.rs

```diff
@@ -69,12 +73,14 @@ pub struct BigTableDbSettings {
     #[serde(deserialize_with = "deserialize_opt_u32_to_duration")]
     pub database_pool_connection_ttl: Option<Duration>,
     /// Max idle time(in seconds) for a connection
-    #[serde(default)]
```

Comment:
Probably removed mistakenly? (it's causing the test failures)

Reply:
yeah, possibly due to the merge. (Sigh)